setup.py

```diff
@@ -28,7 +28,7 @@
     url=URL,
     packages=find_packages(),
     install_requires=['numpy', 'pandas', 'matplotlib',
-                      'click', 'joblib'],
+                      'click', 'joblib', 'scipy'],
```

Comment:
don't do this. It should be a dependency of the solver.

Reply:
so where shall I put the scipy requirement for the nnls_scipy.py to be able to run?